src/trace_reader.rs

```diff
@@ -131,6 +139,7 @@ pub fn collect_samples_from_trace(trace: &CallTrace) -> Vec<Sample> {
 pub struct ResourcesKeys {
```

Comment:
I know it has not been introduced by you, but what this structure stores? I don't understand from the name.

Reply:
It introduces the keys used to create the resource list needed by pprof

Reply:
what are 
> keys used to create the resource list needed by pprof

Reply:
https://github.com/google/pprof/tree/main/proto#measurement-values

Reply:
Yes, but why do we have a structure with three fields without a single hash set? Aren't those fields isomorphic?

Reply:
https://github.com/software-mansion/cairo-profiler/pull/38